doc/doxygen.conf.in

```diff
@@ -1,6 +1,14 @@
 EXAMPLE_PATH = examples
 INPUT += examples/imageDisplay.ipynb
-EXCLUDE += include/lsst/afw/detection/FootprintArray.cc
 EXCLUDE += include/lsst/afw/detection/detail/dgPsf.cc
-EXCLUDE_PATTERNS += */src/*/*.cc
```

Comment:
An alternative I came up with while digging into `astshim`:

    EXCLUDE_PATTERNS += */afw/src/*

Should not match anything from another package.